src/utils/index.ts

```
@@ -230,3 +230,31 @@ export function formatDate(
       });
   }
 }
+
+/**
+ * Formats a number for input fields, handling floating-point precision issues
+ */
+export function formatNumberForInput(value: number, decimals: number = 2): string {
+  if (isNaN(value) || !isFinite(value) || value === null || value === undefined) return '';
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)